dev/cnf/oss_dependencies.maven

```diff
@@ -338,7 +338,6 @@ org.codehaus.woodstox:woodstox-core-asl:4.2.0
 org.cryptacular:cryptacular:1.2.4
 org.eclipse.birt.runtime.3_7_1:org.apache.xml.serializer:2.7.1
 org.eclipse.birt.runtime:org.w3c.css.sac:1.3.1.v200903091627
-org.eclipse.jdt.core.compiler:ecj:4.3.1
```

Comment:
Do any similar updates need to be done in the Closed Liberty side?

Reply:
There are lines for jdt in this file on the closed side so you'll need to look at those.

Reply:
Spoke with Jared. Removing ECJ here shouldn't have any bearing on CL.  However, I created CL issue #24730 since it might be worthwhile to update to a new version, especially if we're supporting Java 17 soon.